### PR TITLE
#[156913221] added circleci config.yml

### PR DESCRIPTION
**Pull request by zenjutahi**
#156913221 Integrate Circle CI for continuous integration and link it with Heroku

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            pipenv run "python manage.py test"
+      - store_test_results:
+          path: test-results
+      - store_artifacts:
+          path: test-results
+          destination: tr1
+
+    deployment:
+      production:
+        branch: develop
```
Comment (jimmykamau) [inline .circleci/config.yml]: Change this to master

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        heroku:
+          appname: hc-bulls
```
Comment (jimmykamau) [inline .circleci/config.yml]: Change this to hc-bullss

**Review by jimmykamau — CHANGES_REQUESTED** [review]
`production` configuration is pointing to the wrong app

**Review by jimmykamau — CHANGES_REQUESTED** [review]
Deployment also won't work until you add the configuration described here: https://circleci.com/docs/2.0/deployment-integrations/#heroku
I've already set the API_KEY and APP_NAME variables.

**Comment by zenjutahi** [comment]
I setup the deployment job

**Review by Bonifase — APPROVED** [review]
Great job! All check are passing, I hope you have configured it with Heroku

**Comment by Bonifase** [comment]
@zenjutahi , I will need your assistance on how to fix the circleci badge in order to do away with the fails.

**Review by ghost — APPROVED** [review]

**Review by jimmykamau — APPROVED** [review]

**Pull request merged**